site/docs/components/landing/NftMintCardDemo.tsx

```diff
@@ -0,0 +1,60 @@
+import { NFTMintCardDefault } from '@coinbase/onchainkit/nft';
+import { WalletDefault } from '@coinbase/onchainkit/wallet';
+import type { ReactNode } from 'react';
+import type { Address } from 'viem';
+import { useAccount } from 'wagmi';
+import App from '../App.tsx';
+
+export const nftMintCardDemoCode = `
+  import { useAccount } from 'wagmi';
+  import { NFTMintCardDefault } from '@coinbase/onchainkit/nft';
+  import { WalletDefault } from '@coinbase/onchainkit/wallet';
+
+  function NftMintCardDemo() {
+    const { address } = useAccount();
+    return (
+      <>
+        {address ? (
+          <NFTMintCardDefault
+            contractAddress="0xed2f34043387783b2727ff2799a46ce3ae1a34d2"
+            tokenId="2"
+          />
```

Comment:
should just be able to include NFTMintCardDefault regardless of address, mint button should render connect wallet if no address

Reply:
confirmed and changed